### PR TITLE
fix(force_auth): make sure force_auth supports the Firefox password manager

### LINKED ISSUE #3049: "No username" in password manager during 'force_auth'
### Actual
image

### Expected

Username should be the email

### Notes

Looks like the fix for  https://github.com/mozilla/fxa-content-server/issues/2754 did not work for force_auth or was not applied

### PR DESCRIPTION
**Pull request by vladikoff**
Fixes #3049

### CODE REVIEW

**Comment by vladikoff** [comment]
@shane-tomlinson r?

File: app/scripts/templates/force_auth.mustache [hunk 1]
```diff
@@ -14,7 +14,7 @@
     </div>
     <form novalidate>
       <p class="prefill">{{ email }}</p>
-
+      <input type="email" class="email hidden" value="{{ email }}" />
```
Comment (shane-tomlinson) [inline app/scripts/templates/force_auth.mustache]: Would it behoove us to add the `disabled` attribute to give some semantic indication that this input cannot be edited?

**Comment by shane-tomlinson** [comment]
screen shot 2015-09-16 at 13 07 52

Tests well, and looks great! Thanks @vladikoff. The `disabled` attribute is not used in the `sign_in` template, so no need. I'm running the functional tests locally and have given travis a kick. I will merge if all the tests pass.

**Comment by shane-tomlinson** [comment]
Tests all pass locally with https://github.com/mozilla/fxa-content-server/pull/3067 applied, travis on the other hand, has issues. I cleared the travis cache and restarted the build.

**Comment by vladikoff** [comment]
Rebased.

**Pull request merged**